apps/src/templates/studioHomepages/JoinSection.jsx

```diff
@@ -43,8 +45,18 @@ const styles = {
     borderWidth: 1,
     borderColor: 'red'
   },
+  wordBoxRTL: {
+    width: styleConstants['content-width'] - 475,
+    marginRight: 25,
+    marginTop: 25,
+    marginBottom: 25,
+    float: 'left',
+    borderWidth: 1,
+    borderColor: 'red'
+  },
```

Comment:
since `wordBox` and `wordBoxRTL` share most of their styles, should `wordBoxRTL` override/add properties on top of `wordBox` (i.e., extend the styles rather than separating them)? then the usage in `render` below could look something like...

```javascript
const wordBoxStyle = {...styles.wordBox, ...(isRtl && styles.wordBoxRTL)}
```

Reply:
Great idea, I'll modify

Reply:
Changes pushed in 91742e9